### PR TITLE
✨ Add Geist font

### PR DESCRIPTION
**Pull request by fayez-nazzal**
The Geist font is becoming increasingly popular. It would be great to be able to use it in other frameworks than Next.js without the hassle of writing custom font faces, which is why I love this to be a part of the fontsource package.

### CODE REVIEW

**Review by ayuhito — CHANGES_REQUESTED** [review]
Hi @fayez-nazzal, thanks for the PR! Apologies for the delay in reviewing it.

I think it would be appropriate for us to regenerate this package with the naming `geist-sans` and `Geist Sans` since there are multiple variants of this font. Otherwise, this looks good to me to merge!

File: geist/200.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/* geist-latin-200-normal */
+@font-face {
+  font-family: 'Geist Sans';
+  font-style: normal;
+  font-display: swap;
+  font-weight: 200;
+  src: url(./files/geist-latin-200-normal.woff2) format('woff2'), url(./files/geist-latin-200-normal.woff) format('woff');
```
Comment (ayuhito) [inline geist/200.css:7]: Fontsource occasionally regenerates the whole repository from scratch and the custom packager relies on the font file names to derive the name of the package. We need to rename the font files for this to not cause any future issues.

**Review by ayuhito — CHANGES_REQUESTED** [review]
Just another change needed before we can merge this!

File: geist/200.css (hunk 1 above)
Comment (fayez-nazzal) [reply geist/200.css:7]: Should the naming follow this pattern?
 `geist-sans-latin-[weight]-normal.woff2`

Comment (ayuhito) [reply geist/200.css:7]: Yup! That would be perfect.

Comment (fayez-nazzal) [reply geist/200.css]: @ayuhito Updated.

Comment (ayuhito) [reply geist/200.css]: Sorry for not noticing this sooner. I think we need to also rename the folder to geist-sans 😓 Hopefully that will the last thing to address.

Comment (fayez-nazzal) [reply geist/200.css]: No worries @ayuhito, I should have spotted it. Can't wait to see the package!

Comment (ayuhito) [reply geist/200.css]: Unfortunately, I just realised we are missing another thing 😓 The `geist-sans` folder is in the root of the monorepo. Can you move the directory into `fonts/other`?

Comment (fayez-nazzal) [reply geist/200.css]: @ayuhito Moved.

**Review by ayuhito — APPROVED** [review]
Thanks for the PR and dealing with all the nitpicks! 🎉 You should see this uploaded to NPM soon and the website search indexes should follow suit in a few hours.

**Pull request merged**